composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": "^7.0 || ^8.0",
+        "php": "^7 || ^8",
```

Comment:
i guess you closed the merge request because you realized this already, but for reference: `^` means that version or any newer minor/patch version. so this change would not change anything.